appstudio-controller/controllers/appstudio.redhat.com/environment_controller_test.go

```diff
@@ -183,6 +192,11 @@ var _ = Describe("Environment controller tests", func() {
 							APIURL:                     "https://my-api-url",
 							ClusterCredentialsSecret:   secret2.Name,
 							AllowInsecureSkipTLSVerify: allowInsecureSkipTLSVerifyParam,
+							ClusterResources:           true,
+							Namespaces: []string{
+								"namespace-1",
+								"namespace-2",
+							},
```

Comment:
Same as above, it would be good to have a update test that continues to verify the original case: cluster resources = false, and empty namespaces.

Reply:
Agreed, I'll make the suggested change.